Add custom metric section for Ruby (#19543)
DataDog/documentation: content/en/continuous_integration/tests/ruby.md

To create filters or `group by` fields for these tags, you must first create facets. For more information about adding tags, see the [Adding Tags][5] section of the Ruby custom instrumentation documentation.

### Adding custom metrics to tests

> Just like tags, you can add custom metrics to your tests by using the current active span:

Reviewer: ```suggestion
Like tags, you can add custom metrics to your tests by using the current active span:
```

```ruby
require 'ddtrace'

# inside your test
Datadog::Tracing.active_span&.set_tag('memory_allocations', 16)
# test continues normally
# ...
```

> Read more about custom metrics in the [Add Custom Metrics Guide][7].

Reviewer: ```suggestion
For more information on custom metrics, see the [Add Custom Metrics Guide][7].
```
